cronsul

```diff
@@ -39,9 +52,8 @@ esac
 
 command=("$@")
 
-# This will be wrong if cron starts our job really late.
-schedule_date="$(( ($(date '+%s') / period) * period ))"
+current_run="$(date '+%s')"
```

Comment:
This change to the scheduling logic means that when a job runs, it inhibits activity for a full period after it has succeeded. This can lead to missed runs. Consider a cronsul job scheduled to run on three nodes at the top of the hour. The cluster is heavily loaded, so the winning job runs at 06:00:05. An hour later, the cluster is less loaded, so all three nodes run cronsul at 07:00:01. None of the jobs runs.

To fix this, I would return to fixed period scheduling, which I think I prefer in any case.